Tests/Editor/Deployment/DeploymentSettingsTests.cs

```diff
@@ -1385,6 +1405,30 @@ private static void SetUpcoreApiforRefreshStackStatusSuccess(Mock<CoreApi> coreA
 
             return underTest;
         }
+        
+        private static DeploymentSettings SetUpDeleteDeployment(Mock<CoreApi> coreApiMock = null)
+        {
+            const string testGameName = "test";
+            const string testProfileName = "testProfile";
+            const string testRegion = "eu-west-1";
+
+            coreApiMock = coreApiMock ?? new Mock<CoreApi>();
+            coreApiMock.Setup(target => target.GetStackName(It.IsAny<string>()))
+                .Returns(testGameName)
+                .Verifiable();
+            coreApiMock.Setup(target => target.GetSetting(It.IsAny<SettingsKeys>()))
+                .Returns(Response.Ok(new GetSettingResponse()))
+                .Verifiable();
+            
+            var stateManagerMock = new Mock<StateManager>(coreApiMock.Object);
+
+            stateManagerMock.SetupGet(l => l.ProfileName).Returns(testProfileName);
+            stateManagerMock.SetupGet(l => l.Region).Returns(testRegion);
```

Comment:
It appears this requires the change in [PR 118](https://github.com/aws/amazon-gamelift-plugin-unity/pull/118/files#r1350264188), if that's the case these should not be in separate PRs, see comment in bootstrap settings test

Reply:
Reverted it from other PR, added back to this PR.